src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/ProjectTemplate.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <!-- Temporary, until implicitly included -->
+  <Import Project="Microsoft.NETCore.props" /> 
+  
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Let's make this the default and include it in the base template. Libraries need to express the output type is library